### PR TITLE
The round ending on one server will send a news report to the other server

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Rounds ending on one server will send a news report to the other server.
/:cl:

Right now they're all from Nanotrasen News Network and I tried to make them vaguely propaganda like, but if people want to write better ones that'd be fantastic.

Will always be easy to add sending reports from other news networks (ruskie radio? syndicate declarations?) if people want to in the future.

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -589,3 +597,48 @@ var/datum/subsystem/ticker/ticker
...
+		if(BLOB_NUKE)
+			news_message = "[station_name()] is currently undergoing decontanimation after a controlled burst of radiation was used to remove a biological ooze. All employees were safely evacuated prior, and are enjoying a relaxing vacation."
+		if(BLOB_DESTROYED)
+			news_message = "[station_name()] is currently undergoing decontamination procedures after the destruction of a biological hazard. As a reminder, any crew members experiencing cramps or bloating should report immediatley to security for incineration."
+		if(CULT_ESCAPE)
+			news_message = "Security Alert: A group of religous fanatics have escaped from [station_name()]."
+		if(CULT_FAILURE)
+			news_message = "Following the dismantling of a restricted cult aboard [station_name()], we would like to remind all employees that worship outside of the Chapel is strictly prohibited, and cause for termination."
+		if(CULT_SUMMON)
+			news_message = "Company officials would like to clarify that [station_name()] was scheduled to be decommissioned following meteor damage earlier this year. Earlier reports of an uknowable eldritch horror were made in error."
```
Comment (PKPenguin321) [inline code/controllers/subsystem/ticker.dm]: unknowable

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -589,3 +597,48 @@ var/datum/subsystem/ticker/ticker
...
+		if(NUKE_MISS)
+			news_message = "The Syndicate have bungled a terrorist attack [station_name()], detonating a nuclear weapon in empty space near by."
```
Comment (PKPenguin321) [inline code/controllers/subsystem/ticker.dm]: Nearby is one word

**Comment by PKPenguin321** [comment]
Neat!

File: code/controllers/subsystem/ticker.dm [hunk 3]
```diff
@@ -589,3 +597,48 @@ var/datum/subsystem/ticker/ticker
...
+		if(GANG_LOSS)
+			news_message = "Organized crime aboard [station_name()] has been stamped out by members of our ever vigilant security team. Remember to thank your assigned officers today!"
+		if(GANG_TAKEOVER)
+			news_message = "Contact with [station_name()] has been lost after a sophisticated hacking attack by organized criminal elements. Stay vigilant!"
+		if(BLOB_WIN)
+			news_message = "[station_name()] was overcome by an unknown biological outbreak, killing all crew on board. Don't let it happen to you! Remember, a clean work station is a safe work station."
+		if(BLOB_NUKE)
+			news_message = "[station_name()] is currently undergoing decontanimation after a controlled burst of radiation was used to remove a biological ooze. All employees were safely evacuated prior, and are enjoying a relaxing vacation."
+		if(BLOB_DESTROYED)
+			news_message = "[station_name()] is currently undergoing decontamination procedures after the destruction of a biological hazard. As a reminder, any crew members experiencing cramps or bloating should report immediatley to security for incineration."
```
Comment (kevinz000) [inline code/controllers/subsystem/ticker.dm]: immediately*

File: code/controllers/subsystem/ticker.dm [hunk 4]
```diff
@@ -589,3 +597,48 @@ var/datum/subsystem/ticker/ticker
...
+		if(CULT_ESCAPE)
+			news_message = "Security Alert: A group of religous fanatics have escaped from [station_name()]."
```
Comment (swindly) [inline code/controllers/subsystem/ticker.dm]: religious

File: code/controllers/subsystem/ticker.dm [hunk 5]
```diff
@@ -589,3 +597,48 @@ var/datum/subsystem/ticker/ticker
...
+		if(OPERATIVE_SKIRMISH)
+			news_message = "A skirmish between security forces and Syndicate agents aboard [station_name()] ended with both sides bloodied but intact."
+		if(REVS_WIN)
+			news_message = "Company officials have reassured investors that despite a union led revolt aboard [station_name()] that there will be no wage increases for workers."
+		if(REVS_LOSE)
+			news_message = "[station_name()] quickly put down a misguided attempt at mutiny. Remember, unionizing is illegal!"
+		if(WIZARD_KILLED)
+			news_message = "Tensions have flared with the Wizard's Federation following the death of one of their members aboard [station_name()]."
+		if(STATION_NUKED)
+			news_message = "[station_name()] activated it's self destruct device for unknown reasons. Attempts to clone the Captain so he can be arrested and executed are under way."
```
Comment (swindly) [inline code/controllers/subsystem/ticker.dm]: its

**Comment by FantasticFwoosh** [comment]
STATION FOR SALE 50% OFF.

File: code/controllers/subsystem/ticker.dm [hunk 6]
```diff
@@ -589,3 +597,48 @@ var/datum/subsystem/ticker/ticker
...
+
+
+/datum/subsystem/ticker/proc/send_news_report()
+	var/news_message
+	var/news_source = "Nanotrasen News Network"
+	switch(news_report)
+		if(NUKE_SYNDICATE_BASE)
+			news_message = "In a daring raid, the heroic crew of [station_name()] detonated a nuclear device in the heart of a terrorist base."
+		if(STATION_DESTROYED_NUKE)
+			news_message = "We would like to reassure all employees that the reports of a Syndicate backed nuclear attack on [station_name()] in fact a hoax. Have a secure day!"
```
Comment (ExcessiveUseOfCobblestone) [inline code/controllers/subsystem/ticker.dm]: is, in fact, just a hoax [maybe]

**Comment by Aloraydrel** [comment]
Did you make sure this is all canon?

File: code/controllers/subsystem/ticker.dm (hunk 6 above)
Comment (Supermichael777) [inline code/controllers/subsystem/ticker.dm]: were in fact
or 
are in fact

File: code/controllers/subsystem/ticker.dm (hunk 2 above)
Comment (Supermichael777) [inline code/controllers/subsystem/ticker.dm]: attack on

**Comment by KingPhilipIII** [comment]
Nanotrasen News Network. 

Station such and such has been destroyed by a nuclear device

**Comment by ChangelingRain** [comment]
If these are sent to a server that isn't actively playing, ie, is starting or ending, they should be delayed until the next roundstart(maybe a bit after) so they're actually IC info.

**Comment by optimumtact** [comment]
Please finish Kor this is really coooooooool

**Comment by KorPhaeron** [comment]
I've only been on for two round endings since I opened this PR, and it only worked in one of them, so I'm not sure what is or isn't working with it yet

**Comment by KorPhaeron** [comment]
@optimumtact 

Actually this would probably be much easier to iron out the bugs with if it was just on the server instead of trying to make sure both servers have this at once with ~8 people having access to update them. It definitely works at least some of the time, I probably just assigned one of the defines wrong somewhere for a roundtype.

**Comment by Jordie0608** [comment]
Sounds like a way to skip out on fixing the bugs ( ͡° ͜ʖ ͡°)

**Pull request merged**